src/sdx/pce/heuristic/heur.py

```diff
@@ -0,0 +1,274 @@
+import argparse
```

Comment:
Would be nice to have some module level docstring to note what this module is supposed to do.